pallets/grants/src/migrations.rs

```diff
@@ -0,0 +1,203 @@
+/*
+ * This file is part of the Nodle Chain distributed at https://github.com/NodleCode/chain
+ * Copyright (C) 2020-2022  Nodle International
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+pub mod v1 {
+	use crate::{Config, Pallet, VestingScheduleOf, VestingSchedules};
+	use frame_support::{
+		dispatch::GetStorageVersion,
+		pallet_prelude::PhantomData,
+		storage::migration::storage_key_iter,
+		traits::{Get, OnRuntimeUpgrade},
+		weights::Weight,
+		Blake2_128Concat, BoundedVec,
+	};
+	use sp_runtime::traits::Saturating;
+	use sp_std::convert::TryInto;
+	use sp_std::vec::Vec;
+
+	pub struct MigrateToBoundedVestingSchedules<T>(PhantomData<T>);
+	impl<T: Config> OnRuntimeUpgrade for MigrateToBoundedVestingSchedules<T> {
+		fn on_runtime_upgrade() -> Weight {
+			let current = Pallet::<T>::current_storage_version();
+			let onchain = Pallet::<T>::on_chain_storage_version();
+
+			log::info!(
+				"on_runtime_upgrade[{:#?}]=> Running migration with current storage version {:?} / onchain {:?}",
+				line!(),
+				current,
+				onchain
+			);
+
+			if current == 1 && onchain == 0 {
+				let pallet_prefix: &[u8] = b"Vesting";
+				let storage_item_prefix: &[u8] = b"VestingSchedules";
+
+				// Check number of entries, and set it aside in temp storage
+				let stored_data: Vec<_> =
+					storage_key_iter::<T::AccountId, Vec<VestingScheduleOf<T>>, Blake2_128Concat>(
+						pallet_prefix,
+						storage_item_prefix,
+					)
+					.collect();
+
+				let mut translated = 0u64;
+				let mut max_schedules: usize = 0;
+
+				assert!(stored_data.len() > 0);
```

Comment:
```suggestion
				assert!(!stored_data.is_empty());
```

Reading this warning in full and it makes sense

```
Warning:   --> pallets/grants/src/migrations.rs:61:13
   |
61 |                 assert!(stored_data.len() > 0);
   |                         ^^^^^^^^^^^^^^^^^^^^^ help: using `!is_empty` is clearer and more explicit: `!stored_data.is_empty()`
   |
   = note: `#[warn(clippy::len_zero)]` on by default
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#len_zero
```

Reply:
Done